src/components/Pagination/Pagination.svelte

```diff
@@ -22,6 +22,13 @@
   import CaretRight24 from 'carbon-icons-svelte/lib/CaretRight24';
   import { cx, fillArray } from '../../lib';
   import Select, { SelectItem } from '../Select';
+  import { afterUpdate, createEventDispatcher } from 'svelte';  
+
+  const dispatch = createEventDispatcher();
+
+  afterUpdate(() => {
+    dispatch('update', {pageSize, page});
```

Comment:
After an initial dispatch, the values for `pageSize` and `page` are converted to a string by the Select component. See the console log screenshot below.

Could you convert the value type to a number/integer when dispatching? (i.e. `Number(pageSize)`)

<img width="214" alt="Screen Shot 2020-05-29 at 4 44 23 PM" src="https://user-images.githubusercontent.com/10718366/83313659-0bb47e80-a1cc-11ea-8141-616549aa6d44.png">

Reply:
OK, I changed it to this:
```js
dispatch('update', {pageSize: parseInt(pageSize), page: parseInt(page)});
```
since they should both always be integers